src/main/kotlin/net/maizegenetics/phgv2/cli/ExportHvcf.kt

```diff
@@ -0,0 +1,62 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import org.apache.logging.log4j.LogManager
+import java.io.File
+
+/**
+ * This will export the give samples dataset to a hvcf file.
+ */
+class ExportHvcf : CliktCommand() {
+
+    private val myLogger = LogManager.getLogger(ExportHvcf::class.java)
+
+    val dbpath by option(help = "Folder name where TileDB datasets are stored")
+        .required()
+
+    val sampleNames by option(help = "Sample names to export")
+        .required()
+
+    val outputDir by option("-o", "--outputDir", help = "Directory where temporary and final files will be written")
+        .required()
+
+    override fun run() {
+
+        // This is the tiledbvcf command we want to run:
+        // Doing this with a ProcessBuilder and using the phg_v2 conda environment
+        // tiledbvcf export --uri tiledb/hvcf_dataset -O z --sample-names Ref --output-dir exported-vcfs
```

Comment:
The comment should show "-O v" as that is what you are sending.  You might also mention the output will be vcf file format

Reply:
Ok, I changed that